src/styles/index.css

```diff
@@ -0,0 +1,5 @@
+body {
+  margin: 0;
+}
```

Comment:
The CSS file currently only resets the body margin. Ensure that all necessary styles from your SCSS files are compiled and included here, especially since the HTML file references styles that are not present in this CSS file.